src/Badges.sol

```diff
@@ -67,7 +67,7 @@ contract Badges is
     address to,
     string calldata uri,
     bytes calldata signature
-  ) external virtual returns (uint256) {
+  ) external virtual override returns (uint256) {
```

Comment:
Here it wasn't compiling without `override`, This function is defined in the interface, so we need `override`. I'm actually not sure how it was compiling before